libbeat/cmd/instance/beat.go

```diff
@@ -244,12 +244,6 @@ func NewBeat(name, indexPrefix, v string, elasticLicensed bool) (*Beat, error) {
 		return nil, err
 	}
 
-	h, err := sysinfo.Host()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get host information: %w", err)
-	}
-	fqdn := h.Info().FQDN
-
```

Comment:
This piece of code had to be moved to slightly later in the Beat's execution because we need logging to be configured to emit a warning if the FQDN lookup fails.  You'll find it elsewhere in this PR, along with the logging.